payments_config/products.py

```diff
@@ -21,7 +24,23 @@
                 'img': ('https://raw.githubusercontent.com/mozilla'
                         '/payments-config/master/payments_config'
                         '/assets/mortar.png'),
+                'recurrence': 'monthly',
             }
         ]
-    }
+    },
+    'mozilla-foundation': {
+        'email': 'support@concrete.mozilla.org',
+        'name': _('Mozilla Foundation'),
```

Comment:
we could go with 'Mozilla Forest Preserve' if we wanted something less real so that we don't present any misleading amounts or donation levels.